### ISSUE TITLE
Last remaning Bhaflau Remnants floor 4 Archaic Gear warped after being hit with Bloody Bolt at 25yalms

### ISSUE THREAD

**Issue by Avarghaladion** [EdenServer/community#5865]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted.
- [X] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [X] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
30181205_0

After safely clearing 9/10 floor 4-east gears in Bhaflau Remnants, the 10th gear immediately warped after being shot at exactly 25 yalms with a Bloody Bolt. The gear was stationary throughout the shot, which was made by a THF/NIN using Ziska's Crossbow. No wind weather was present, no other party member was within 40 yalms of the gear, and the THF timed their movement such that the shot went off as they were taking their first steps way from the mob back towards the group.

The gear in question was the southwesternmost gear that spawns in H/I-7 just before the Rampart room. At the time of the shot, the gear was near the southwest corner of the room, the THF was at max /shoot range between the gear and the hallway, and the rest of the party was on the stairs in the hallway or further back. 

I have not been able to find any indication of this happening in era. FFXI Wiki and bgwiki both indicate that damaging spells and abilities cause this but that ranged attacks are unequivocally safe. I'm guessing the gear is perceiving the additional effect as a damaging spell/ability and that any bolt/arrow with an additional damage effect might cause this on Eden.

**Comment by DoughnutEdenXI** [comment]
If the bloody bolt procced that's a magic damage hit

**Comment by Avarghaladion** [comment]
Yeah I just can't find any evidence of that happening in era so far. I can see the logic of both cases but does seem odd in that the proc can only occur if the non-magical potion of the attack connects. Ie. it should be seeing itself as having been pulled due to the shot itself when the proc occurs, no?
